src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -71,12 +72,39 @@ class NetworkOPsImp final
     : public NetworkOPs
     , public beast::DeadlineTimer::Listener
 {
-public:
-    enum Fault
+    /**
+     * Transaction with input flags and results to be applied in batches.
+     */
+    class TransactionStatus
+    {
+    public:
+        Transaction::pointer transaction;
+        bool admin;
```

Comment:
These parameters are passed in as bools from several locations in the code, and often as callbacks. To define boolean enums for these parameters when the callers don't necessary have access to the NetworkOPs class presents some logistical issues about where to define these enums, whether to define the same enum bool in multiple places, do conversions, and so on.

I don't think this makes things more readable--when I look up some class that simply has the values "true" and "false" or some variation, I think "why bother?"